ui/pages/settings/settings.container.js

```diff
@@ -64,7 +64,7 @@ const mapStateToProps = (state, ownProps) => {
 
   const addressName = getAddressBookEntryName(
     state,
-    isHex(pathNameTail) && isValidAddress(pathNameTail) ? pathNameTail : '',
+    isValidHexAddress(pathNameTail) ? pathNameTail : '',
```

Comment:
I wonder if we have any existing entries that this would be `false` for? :thinking:

Reply:
not sure I understand the question here, this code previously used 'isValidAddress' from ethereumjs-util and as part of the original ethereumjs-util upgrade I added 'isHex' ahead of this because non-address strings in the URL were being checked to see if they were a valid address and failing. Can't remember the exact path that was being checked when I noticed the failure case :/

Reply:
I was thinking of the difference in checksumming, wondering if this change would prevent this property from being set when viewing a contact with a mixed-case, non-checksummed address.

But with the most recent changes, that's no longer a concern.